cypress/integration/smoke.spec.js

```diff
@@ -15,10 +15,14 @@ describe('Smoke Test', () => {
   it('assert the number of boards equal boards in `boards.json`', () => {
     cy.get('div.boardList').as('boardList')
 
-    cy.readFile(boardsPath).then(
-      boards => {
-        assert(boards.length > 100)
-        cy.get('@boardList').children().should('have.length', boards.length)
-      })
+    cy.readFile(boardsPath).then(boards => {
+      assert(
+        boards.length >= 114,
+        `not enough boards in boards.json, expecting at least 114 got ${boards.length}`
```

Comment:
Shouldn't the `114` be an environment variable? :rofl:

Reply:
Haha, maybe. I am actually mostly trying to figure out why this test passes when boards.json appears to be empty. https://travis-ci.org/github/kitspace/kitspace/builds/735101062#L775

Reply:
The Cypress [video](https://dashboard.cypress.io/projects/pk2d9c/runs/129/specs/56283a73-44e5-41f1-a967-b906c0e824eb/video) indicates the boards are rendered on the page.
Not tested hypothesis: the boards not showing in the Travis log has something to do with the way they handle the cat command itself.

Reply:
Also the `makeBoardsJson` task [runs](https://travis-ci.org/github/kitspace/kitspace/builds/735101062#L767).

Reply:
You are right. Somehow `echo "$(cat ...` does the trick. I think it might be because these json files don't have a newline at the end of the file?